recipes/selectfasta/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set name = "selectFasta" %}
+{% set version = "1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/andvides/selectFasta/archive/v1.0.tar.gz
+  sha256: 4163e520e87708508999f4d73da6ccf6ef9b7cd2
+  
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    
+test:
+  commands:
+    - selectFasta -h
+
+about:
+  home: https://github.com/andvides/selectFasta/
+  license: GNU
```

Comment:
You will need to add proper license file/information in the repo.

Reply:
done